Documentation/operations/metrics.rst

```diff
@@ -286,146 +285,145 @@ report the eBPF memory usage directly managed by Cilium.
-======================================== ================================================== ========================================================
-``event_ts``                             ``source``                                         Last timestamp when we received an event
-======================================== ================================================== ========================================================
+======================================== ================================================== ========== ========================================================
+Name                                     Labels                                             Default    Description
+======================================== ================================================== ========== ========================================================
+``event_ts``                             ``source``                                         Enabled    Last timestamp when we received an event
+======================================== ================================================== ========== ========================================================
 
 Controllers
 ~~~~~~~~~~~
 
-======================================== ================================================== ========================================================
-Name                                     Labels                                             Description
-======================================== ================================================== ========================================================
-``controllers_runs_total``               ``status``                                         Number of times that a controller process was run
-``controllers_runs_duration_seconds``    ``status``                                         Duration in seconds of the controller process
-======================================== ================================================== ========================================================
+======================================== ================================================== ========== ========================================================
+Name                                     Labels                                             Default    Description
+======================================== ================================================== ========== ========================================================
+``controllers_runs_total``               ``status``                                         Enabled    Number of times that a controller process was run
+``controllers_runs_duration_seconds``    ``status``                                         Enabled    Duration in seconds of the controller process
+======================================== ================================================== ========== ========================================================
```

Comment:
It seems like controllers_failing is missing here, could you add it as well?

Reply:
It is added in `Clusters` #L219 section. Didn't seem right that time as well. Will correct its place.